snap/snapcraft.yaml

```diff
@@ -0,0 +1,54 @@
+name: falltergeist # you probably want to 'snapcraft register <name>'
+version: 0.3.1 # just for humans, typically '1.2+git' or '1.3.2'
+summary: Open Source Fallout 2 game engine # 79 char long summary
+description: |
+  Opensource crossplatform Fallout 2™ engine writen in C++ and SDL. 
+  
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: devmode # use 'strict' once you have the right plugs and slots
+
+apps:
+ falltergeist:
+  command: falltergeist.launcher
+  desktop: share/applications/falltergeist.desktop
+  plugs:
+  - a1sa
```

Comment:
`a1sa`? Isn't that some typo?

Reply:
Yes... :blush: That explains the warning at install. I'm not 100% sure its needed